build-scripts/build.sh

```diff
@@ -192,7 +192,7 @@ else
     export UBSAN_OPTIONS=print_stacktrace=1
     if [ "$TRAVIS_OS_NAME" == "osx" ] || [ "$OS" == "macos-10.15" ]
     then
-        run_tests ./tests/cata_test
+        parallel --tag --verbose --linebuffer ./tests/cata_test --min-duration 0.2 --use-colour yes --rng-seed time ::: "crafting_skill_gain" "[slow] ~crafting_skill_gain" "~[slow] ~[.]"
     else
         run_tests ./tests/cata_test &
         pids[0]=$!
```

Comment:
If this is not used on macs, then `wait -n` can be used again, I would think.

Reply:
That makes sense. But for the scope of this PR, I'm only going to add GNU Parallel to build script first.